vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/JsonResponseValidationSteps.java

```diff
@@ -372,6 +379,11 @@ private int getElementsNumber(String json, String jsonPath)
 
     private Optional<List<?>> getElements(String json, String jsonPath)
     {
+        if (json == null)
```

Comment:
when could it happen?

Reply:
When bad request result in client-side error response. Will remove